src/connector/src/sink/clickhouse.rs

```diff
@@ -190,18 +190,9 @@ impl ClickHouseEngine {
     }
 }
 
-const POOL_IDLE_TIMEOUT: Duration = Duration::from_secs(5);
```

Comment:
Is is ok to remove this `POOL_IDLE_TIMEOUT` setting? I guess the original purpose not to use `ClickHouseClient::default()` is to set this config. cc @xxhZs

Reply:
The corresponding pr is this one, https://github.com/risingwavelabs/risingwave/pull/12041/files
 the default value of `POOL_IDLE_TIMEOUT` is 2s in clickhouse.rs,  the reason for not using the default here is to support https urls

Reply:
If so, we may not modify the code here? @BugenZhao

Reply:
May need to run a test to connect to clickhouse with https. If it passes, we can then sync the change.

Reply:
Cherry-picked the fix into our fork of `clickhouse.rs`.